cli/pkg/kctrl/cmd/app/release/app_spec_builder.go

```diff
@@ -33,6 +33,7 @@ type AppSpecBuilderOpts struct {
 	BuildExport   []appinit.Export
 	BundleImage   string
 	Debug         bool
+	Tag           string
```

Comment:
nit: Tag => BundleTag (in line with how we use `BundleImage`)

Reply:
Adopted